modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -627,6 +635,42 @@ private long _getFileEntryTypeId() {
 		return ParamUtil.getLong(_httpServletRequest, "fileEntryTypeId", -1);
 	}
 
+<<<<<<< HEAD
+=======
```

Comment:
Please make sure rebases are clean. This should never happen, even if it is fixed in a following commit.

Reply:
True, I'm sorry I made this mistake